src/coreclr/tools/aot/ILCompiler.LLVM/Compiler/LLVMCodegenCompilationBuilder.cs

```diff
@@ -87,6 +87,8 @@ internal void SetOptions(IEnumerable<KeyValuePair<string, string>> options)
         public CorInfoLlvmEHModel ExceptionHandlingModel { get; private set; } = CorInfoLlvmEHModel.Wasm;
 
         // Below options are debug-only and not supported.
-        public int MaxLlvmModuleCount { get; private set; } = 8;
+
+        // Set to a high number to reduce the memory use of clang.
+        public int MaxLlvmModuleCount { get; private set; } = 16;
```

Comment:
```suggestion
        public int MaxLlvmModuleCount { get; private set; } = 16;
```
I don't think we need this comment - 16 is not an exceptionally high number.